server/cluster/cluster.go

```diff
@@ -1369,12 +1371,22 @@ func (c *RaftCluster) GetOfflineRegionStatsByType(typ statistics.RegionStatistic
 	return c.regionStats.GetOfflineRegionStatsByType(typ)
 }
 
-func (c *RaftCluster) updateRegionsLabelLevelStats(regions []*core.RegionInfo) {
+func (c *RaftCluster) updateStats(region *core.RegionInfo) {
```

Comment:
I think we should handle region in batches. Lock and Unlock for every region could be slow.

Reply:
good idea